lib/ReactViews/BottomDock/Timeline/DateTimePicker.tsx

```diff
@@ -0,0 +1,752 @@
+                }
+              >
+                <span>
+                  {item} : 00 - {item + 1} : 00
+                </span>{" "}
+                <span>
+                  (
+                  {
+                    datesObject[this.currentDateIndice.year!][
+                      this.currentDateIndice.month!
+                    ][this.currentDateIndice.day!][item].length
+                  }{" "}
+                  options)
+                </span>
+              </DateButton>
+            ))}
+          </GridBody>
+        </Grid>
+      );
+    } else {
+      return this.renderList(
+        datesObject[this.currentDateIndice.year][this.currentDateIndice.month][
+          this.currentDateIndice.day
+        ].dates
+      );
+    }
+  }
+
+  renderMinutesView(datesObject: ObjectifiedYears) {
+    if (
+      !isDefined(this.currentDateIndice.year) ||
+      !isDefined(this.currentDateIndice.month) ||
+      !isDefined(this.currentDateIndice.day) ||
+      !isDefined(this.currentDateIndice.hour)
+    ) {
+      return null;
+    }
+    const options =
+      datesObject[this.currentDateIndice.year][this.currentDateIndice.month][
+        this.currentDateIndice.day
+      ][this.currentDateIndice.hour];
+    return this.renderList(options);
+  }
+
+  @action
+  goBack() {
+    if (isDefined(this.currentDateIndice.time)) {
+      if (!isDefined(this.currentDateIndice.month)) {
+        this.currentDateIndice.year = undefined;
+        this.currentDateIndice.day = undefined;
+      }
+
+      if (!isDefined(this.currentDateIndice.hour)) {
+        this.currentDateIndice.day = undefined;
+      }
+
+      if (!isDefined(this.currentDateIndice.day)) {
+        this.currentDateIndice.month = undefined;
+      }
+
+      this.currentDateIndice.hour = undefined;
+      this.currentDateIndice.time = undefined;
+    } else if (isDefined(this.currentDateIndice.hour)) {
+      this.currentDateIndice.hour = undefined;
+    } else if (isDefined(this.currentDateIndice.day)) {
+      this.currentDateIndice.day = undefined;
+    } else if (isDefined(this.currentDateIndice.month)) {
+      this.currentDateIndice.month = undefined;
+    } else if (isDefined(this.currentDateIndice.year)) {
+      this.currentDateIndice.year = undefined;
+    } else if (isDefined(this.currentDateIndice.century)) {
+      this.currentDateIndice.century = undefined;
+    }
+  }
+
+  toggleDatePicker() {
+    if (!this.props.isOpen) {
+      this.props.onOpen();
+    } else {
+      this.props.onClose();
+    }
+  }
+
+  render() {
+    if (this.props.dates) {
+      const datesObject = this.props.dates;
+      return (
+        <div
+          css={`
+            color: #fff;
+            display: table-cell;
+            width: 30px;
+            height: 30px;
+          `}
+          onClick={event => {
+            event.stopPropagation();
+          }}
+        >
+          {this.props.showCalendarButton && (
+            <button
+              css={`
```

Comment:
Add some sort of title - only especially important as the button doesn't have any text in it (it's just an icon as sole child)

```suggestion
            <button
              title={t("diffTool.datePicker.dateButtonTitle")}
              css={`
```